baybe/searchspace/continuous.py

```diff
@@ -42,6 +43,37 @@ class SubspaceContinuous(SerialMixin):
     )
     """List of linear inequality constraints."""
 
+    def __str__(self) -> str:
+        if self.is_empty:
+            return ""
+
+        start_bold = "\033[1m"
+        end_bold = "\033[0m"
+
+        # Convert the lists to dataFrames to be able to use pretty_printing
+        param_dict = list()
```

Comment:
You can create these lists more efficiently using list comprehensions, i.e., via 
`param_dict = [param.summary() for param in self.parameters]``
Also, you should rename this and the following parts to `param_list` or `param_dictionaries` since this makes it clear that the object itself is a list (first case) or that it contains the dictionaries of the parameters (second case)

Reply:
Fixed. Thank you for the suggestion!